deepforest/utilities.py

```diff
@@ -63,43 +91,30 @@ def use_bird_release(save_dir=os.path.join(_ROOT, "data/"), prebuilt_model="bird
     
     if check_release:
         # Find latest github tag release from the DeepLidar repo
-        _json = json.loads(
-            urllib.request.urlopen(
-                urllib.request.Request(
-                    'https://api.github.com/repos/Weecology/BirdDetector/releases/latest',
-                    headers={'Accept': 'application/vnd.github.v3+json'},
-                )).read())
-        asset = _json['assets'][0]
-        url = asset['browser_download_url']
-    
-        # Check the release tagged locally
-        try:
-            release_txt = pd.read_csv(save_dir + "current_bird_release.csv")
-        except BaseException:
-            release_txt = pd.DataFrame({"current_bird_release": [None]})
-    
+        release_txt = check_new_release(save_dir)
+        if release_txt.current_bird_release[0] and update:
         # Download the current release it doesn't exist
-        if not release_txt.current_bird_release[0] == _json["html_url"]:
-    
-            print("Downloading model from BirdDetector release {}, see {} for details".format(
-                _json["tag_name"], _json["html_url"]))
-    
-            with DownloadProgressBar(unit='B',
-                                     unit_scale=True,
-                                     miniters=1,
-                                     desc=url.split('/')[-1]) as t:
-                urllib.request.urlretrieve(url, filename=output_path, reporthook=t.update_to)
-    
-            print("Model was downloaded and saved to {}".format(output_path))
-    
-            # record the release tag locally
-            release_txt = pd.DataFrame({"current_bird_release": [_json["html_url"]]})
-            release_txt.to_csv(save_dir + "current_bird_release.csv")
-        else:
-            print("Model from BirdDetector Repo release {} was already downloaded. "
-                  "Loading model from file.".format(_json["html_url"]))
-    
-        return _json["html_url"], output_path
+            if not release_txt.current_bird_release[0] == _json["html_url"]:
+        
+                print("Downloading model from BirdDetector release {}, see {} for details".format(
+                    _json["tag_name"], _json["html_url"]))
+        
+                with DownloadProgressBar(unit='B',
+                                        unit_scale=True,
+                                        miniters=1,
+                                        desc=url.split('/')[-1]) as t:
+                    urllib.request.urlretrieve(url, filename=output_path, reporthook=t.update_to)
+        
```

Comment:
These are code Clean up changes, make them in a different PR or a different commit.